### PR TITLE
schema fixes

### PR DESCRIPTION
**Pull request by sk91**

### CODE REVIEW

File: schema/attribute/drivers-license.json [hunk 1]
```diff
@@ -4,17 +4,26 @@
 	"identityAttribute": true,
 	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
 	"title": "Drivers License",
-	"description": "A copy of an individuals drivers license.",
+	"description": "Please provide an image of your driver's license. Front and back cover are required.",
```
Comment (felipewer) [inline schema/attribute/drivers-license.json]: This description is confusing. It asks for **an** image of the drivers's license (Indicating a single image) and then goes on to say that front and back are required. I would be left scratching my head and looking for image editors.

A better option could be:

"Please provide images of the front and back of your driver's license."

File: schema/attribute/external-document.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"$id": "http://platform.selfkey.org/schema/attribute/external-document.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"identityAttribute": true,
+	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "External Document",
```
Comment (felipewer) [inline schema/attribute/external-document.json]: Doesn't it need a description?

File: schema/attribute/member-register.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"identityAttribute": true,
+	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "Member Register",
+	"type": "object",
+	"properties": {
+		"images": {
+			"$ref": "http://platform.selfkey.org/schema/file/multi-image.json"
+		},
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/member-register.json]: `issued` ?

File: schema/attribute/member-register.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+	"$id": "http://platform.selfkey.org/schema/attribute/member-register.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"identityAttribute": true,
+	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "Member Register",
```
Comment (felipewer) [inline schema/attribute/member-register.json]: No description?

File: schema/attribute/national-id.json [hunk 5]
```diff
@@ -4,24 +4,29 @@
...
-			"$ref": "http://platform.selfkey.org/schema/document/image.json"
+			"$ref": "http://platform.selfkey.org/schema/file/image.json"
 		},
-		"additional": {
-			"type": "array",
-			"title": "Additional images",
-			"items": {
-				"$ref": "http://platform.selfkey.org/schema/document/image.json"
-			}
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/national-id.json]: `issued` ?

File: schema/attribute/passport.json [hunk 6]
```diff
@@ -4,12 +4,24 @@
...
 	"title": "Passport",
-	"description": "A copy of an individuals passport.",
+	"description": "Please provide an image of your passport. It must still be valid.",
 	"type": "object",
 	"properties": {
 		"image": {
-			"$ref": "http://platform.selfkey.org/schema/document/image.json"
+			"$ref": "http://platform.selfkey.org/schema/file/image.json"
+		},
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/passport.json]: `issued` ?

File: schema/attribute/registration-certificate.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+		},
+		"company_name": {
+			"title": "Compnay Name",
+			"type": "string"
+		},
+		"registration_number": {
+			"title": "Registration Number",
+			"type": "integer"
+		},
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/registration-certificate.json]: `issued` ?

File: schema/attribute/tax-certificate.json [hunk 8]
```diff
@@ -4,11 +4,17 @@
...
 	"title": "Tax Certificate",
-	"description": "A copy of a tax certificate.",
+	"description": "Please provide a copy of a recent tax certificate. No older than 12 months.",
 	"type": "object",
 	"properties": {
 		"image": {
-			"$ref": "http://platform.selfkey.org/schema/document/image.json"
+			"$ref": "http://platform.selfkey.org/schema/file/image.json"
+		},
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/tax-certificate.json]: `issued` ?

File: schema/attribute/utility-bill.json [hunk 9]
```diff
@@ -4,12 +4,18 @@
...
-	"description": "A copy of a utility bill.",
+	"description": "Please provide an image with a recent utility bill. No older than 3 months.",
 	"type": "object",
 	"properties": {
-		"image": {
-			"$ref": "http://platform.selfkey.org/schema/document/image.json"
+		"images": {
+			"$ref": "http://platform.selfkey.org/schema/file/muilti-image.json"
+		},
+		"issue": {
```
Comment (felipewer) [inline schema/attribute/utility-bill.json]: `issued` ?

File: schema/attribute/voice-id.json [hunk 10]
```diff
@@ -4,12 +4,9 @@
...
-	"properties": {
-		"audio": {
-			"$ref": "http://platform.selfkey.org/schema/document/audio.json"
-		}
-	},
-	"required": ["audio"]
+	"description": "Please provide a recording with your voice authentication phrase.",
+	"type": "array",
+	"items": {
+		"$ref": "http://platform.selfkey.org/schema/document/audio.json"
```
Comment (felipewer) [inline schema/attribute/voice-id.json]: Shouldn't the path be changed from `/document/` to `/file/` as per image?

**Comment by sk91** [comment]
@felipewer fixed non description staff, waiting for @oanamangiurea to update descriptions on the other ones

**Comment by sk91** [comment]
@felipewer all updated

**Pull request merged**